conf/bookkeeper.conf

```diff
@@ -694,7 +694,7 @@ httpServerEnabled=false
 
 # The http server port to listen on. Default value is 8080.
 # Use `8000` as the port to keep it consistent with prometheus stats provider
-httpServerPort=8000
+httpServerPort=8001
```

Comment:
@liangyepianzhou If the bookie side enabled the http server, prometheus should also use the http server port, not a separate port. Can you reproduce the issue?

Reply:
Thanks for your feedback!
When deploying the cluster before, there were often conflicts due to the port of prometheusStatsHttpPort=8000.
Then I found that there are two configurations whose port is 8000 in the bookkeeper.
If not for this reason I will redeploy the cluster and find out the real reason for the port conflict later.

Reply:
If you are deploying the bookkeeper and zookeeper on the same node, it should conflict with the port used by Zookeeper